### PR TITLE
connect: example instructions

### PR DESCRIPTION
**Pull request by alyssawilk**
Tweaking connect example so it works both for tunneling and direct CONNECT.
Manually verified instructions in docs/root/intro/arch_overview/http/upgrades.rst still work and the new curl instructions do as well.

### CODE REVIEW

File: configs/terminate_connect.yaml [hunk 1]
```diff
@@ -57,8 +64,3 @@ static_resources:
               socket_address:
                 address: www.google.com
                 port_value: 443
-    transport_socket:
```
Comment (mattklein123) [inline configs/terminate_connect.yaml:60]: I'm guessing you didn't mean to delete this?

/wait

Comment (ns-kvaradhan) [reply configs/terminate_connect.yaml:60]: Hi Alyssa, Thanks for the corrected version,
I just tried it out and it works like a charm,
Kannan

Comment (alyssawilk) [reply configs/terminate_connect.yaml:60]: I did mean to delete it, but I pushed the wrong comments.

Apologies for the force push - I'd fixed the information locally and somehow failed to  update upstream before I sent this out.

**Review by mattklein123 — APPROVED** [review]
Cool makes more sense now!

File: configs/terminate_connect.yaml [hunk 2]
```diff
@@ -56,9 +59,4 @@ static_resources:
             address:
               socket_address:
                 address: www.google.com
-                port_value: 443
-    transport_socket:
-      name: envoy.transport_sockets.tls
-      typed_config:
-        "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
-        sni: www.google.com
+                port_value: 80
```
Comment (ns-kvaradhan) [inline configs/terminate_connect.yaml]: Why does the port value change?

Comment (alyssawilk) [reply configs/terminate_connect.yaml]: You're looking at an outdated version - the version submitted used 443 which should work :-)

**Pull request merged**